requirements.txt

```diff
@@ -1,9 +1,5 @@
 mdutils==1.2.2
-GitPython==3.0.5
-gitdb2==2.0.6
 parameters-validation==1.1.5
 pyspark==3.0.0
-PyYAML==5.3
 cassandra-driver==3.22.0
-sphinxemoji==0.1.6
-typing-extensions==3.7.4.2
\ No newline at end of file
+pandas==1.0.3
```

Comment:
Why you change this file? :thinking: :thought_balloon: 
We need the `sphinxemoji` on requirements, because of the Read the Docs.

Reply:
I moved to `requirements.dev.txt` as it's not necessary to butterfree library